### PR TITLE
fix: center align text when bind to container via context menu

### LINKED ISSUE #6434: bind to container defaults to left-aligned text
Steps:

1. reset settings to default (by resetting the canvas)
2. create reactangle & text elements (separately)
3. select both and bind them together
4. text is left-aligned.

Should default to `center`.

### PR DESCRIPTION
**Pull request by SnorfYang**
- [ ] closes #6434
- [x] Add tests

### CODE REVIEW

File: src/tests/binding.test.tsx [hunk 1]
```diff
@@ -309,4 +309,33 @@ describe("element binding", () => {
     expect(arrow2.startBinding?.elementId).toBe(container.id);
     expect(arrow2.endBinding?.elementId).toBe(rectangle1.id);
   });
+  
+  it("should text be centered when binding to a rectangle", async () => {
```
Comment (ad1992) [inline src/tests/binding.test.tsx:316]: can you move this test to `textwysiwyg.test.tsx` ?, 
All text containers related tests we keep in the above file though can decouple later but for now lets keep it in same place so its easier to decouple later as well.
This file deals with tests related to `binding.ts`

**Review by ad1992 — COMMENTED** [review]
@SnorfYang thanks for the fix ❤️

File: src/tests/binding.test.tsx (hunk 1 above)
Comment (SnorfYang) [reply src/tests/binding.test.tsx:316]: I don't see this file in the test folder. Should I create <code>textwysiwyg.test.tsx</code>?

Comment (SnorfYang) [reply src/tests/binding.test.tsx:316]: Do I really need to move this test? I notice in #6450, a comment says this test should be in <code>binding.test.tsx</code>.

Comment (ad1992) [reply src/tests/binding.test.tsx:316]: Yes please move it to https://github.com/excalidraw/excalidraw/blob/master/src/element/textWysiwyg.test.tsx, thats where we have all the tests related to text containers except for labelled arrows.

Comment (SnorfYang) [reply src/tests/binding.test.tsx]: done.

File: src/element/textWysiwyg.test.tsx [hunk 2]
```diff
@@ -1504,5 +1505,36 @@ describe("textWysiwyg", () => {
...
+        text: "Snorf",
+      });
+
+      h.elements = [rectangle1, text1];
+
+      API.setSelectedElements([rectangle1, text1]);
+
+      expect(h.state.selectedElementIds[(rectangle1.id, text1.id)]).toBe(true);
+
+      h.app.actionManager.executeAction(actionBindText);
```
Comment (ad1992) [inline src/element/textWysiwyg.test.tsx:1529]: Actually we don't need another test for this, we already have one test which checks the whole flow by opening context menu and triggering the action so lets merge it there - https://github.com/excalidraw/excalidraw/blob/master/src/element/textWysiwyg.test.tsx#L743
just add the alignment and coords assertions in the same test

**Review by ad1992 — CHANGES_REQUESTED** [review]

File: src/element/textWysiwyg.test.tsx (hunk 2 above)
Comment (SnorfYang) [reply src/element/textWysiwyg.test.tsx]: Done.

**Review by ad1992 — APPROVED** [review]
LGTM, thanks @SnorfYang ❤️

**Pull request merged**